protocols/mafia2mp.js

```diff
@@ -21,6 +21,7 @@ export default class mafia2mp extends Core {
     state.numplayers = parseInt(this.readString(reader))
     state.maxplayers = parseInt(this.readString(reader))
     state.raw.gamemode = this.readString(reader)
+    state.version = state.raw.gamemode
```

Comment:
![image](https://github.com/gamedig/node-gamedig/assets/7972857/52bf3ade-27d9-4392-b084-2001624093da)
Apparently this is a customizable field, but I've seen that it really is a version on some servers, so I guess it's fine, noting this just for future references.